substrate/zombienet-sdk/tests/lib.rs

```diff
@@ -0,0 +1,4 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+mod staking_next;
```

Comment:
Why have you chosen to have these tests to be in `./substrate` vs in `./substrate/frame/staking-async`? 

Also noting the new name is staking-async, not next.

Reply:
It's a common pattern to have all zombienet tests in `polkadot/zombiened-sdk-tests`, `substrate/zombienet-sdk-tests`, etc. 

But I've got no strong preference on this. If you want, I'll move them to to `./substrate/frame/staking-async`.

I'll fix the name.

Reply:
Both are fixed.